src/App.jsx

```diff
@@ -48,10 +52,8 @@ function App() {
 				},
 			});
 			if (!response.ok) {
-				throw new Error(
-					toast.error(
-						`Errr, something isn't right here. Please reload page. Error: ${response.status}`
-					)
+				toast.error(
```

Comment:
[Throw](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/throw) is not needed in this instance. I also believe that invoking the toast inside of the throw was causing it not to auto-dismiss.